README.md

```diff
@@ -1,6 +1,6 @@
 # Tracee - Container and system tracing using eBPF
 
-Tracee is a lightweight and easy to use container and system tracing tool. It allows you to observe system calls and other system events in real time. A unique feature of Tracee is that it will only trace newly created processes and containers (that were started after Tracee has started), in order to help the user focus on relevant events instead of every single thing that happens on the system (which can be overwhelming).
+Tracee is a lightweight and easy to use container and system tracing tool. It allows you to observe system calls and other system events in real time. A unique feature of Tracee is that it will only trace newly created processes and containers (that were started after Tracee has started), in order to help the user focus on relevant events instead of every single thing that happens on the system (which can be overwhelming). Tracee is lightweight and modular so adding new events to Tracee (especially system calls) is straightforward, and will usually require no more than adding few lines of code.
```

Comment:
> Tracee is lightweight and modular so adding new events...

I would remove the start of this line and just start from "Adding new events...." as we already say Tracee is lightweight in the beginning of this paragraph.

For your consideration...

Reply:
ok, updated